CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 
 ### Removed
 - Removed possibility to export entries/databases to an `.sql` file, as the logic cannot easily use the correct escape logic
+- Removed support of old groups format (used prior to JabRef version 1.6)
```

Comment:
Ah ok. Can you provide a hint in the changelog that one can still use a JabRef version 3.3 or older to read it and convert the group format to a newer one so that it is usable for 3.4+?